vim/core/oni-plugin-cpp/lib/index.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs")
+const path = require("path")
+const childProcess = require("child_process")
+
+const activate = (Oni) => {
+
+    const command = Oni.configuration.getValue("cpp.langServerCommand", "clangd")
+
+    const serverOptions = {
+        command,
```

Comment:
Regarding the `stdErrAsLog` issue - it looks like it's picked up as part of these options, so you'll need to include it as part of this object:

```
const serverOptions = {
   command,
   stdErrAsLog: true,
}
```

I think that should address the issue you had where it wasn't getting set / defined in the language client code